src/styles/partials/components/_breadcrumbs.scss

```diff
@@ -0,0 +1,73 @@
+.dg_breadcrumbs {
+  color: $black;
+  display: inline-block;
+  font-family: $default-heading-font;
+  font-size: $smallest-font-size;
+  font-weight: $font-bold;
+  list-style: none;
+  margin-bottom: 0px;
+  margin-top: $margin-small;
+  padding-bottom: $padding-smallest;
+  padding-left: 0px;
+  padding-top: 0px;
+  text-transform: uppercase;
+
+  i {
+    font-size: $h3-font-size;
+  }
+
+  ul {
+    margin-bottom: 0px;
+    padding-bottom: 0px;
+  }
+
+  li {
+    display: inline;
+
+    a {
+      color: $blue;
+      font-size: $smallest-font-size;
+      font-weight: $font-normal;
+      padding: 0px 30px 0px 0px;
+      text-decoration: none;
+
+      &:hover {
+        color: $vibrant-blue;
+      }
+
+      &:after {
+        color: $gray-light;
+        font-size: $h1-font-size;
+        font-weight: $font-bold;
+        @include icon-content("\f105");
```

Comment:
what do you think about creating varaibles for icons...for example.

$angle-right-icon: "\f105";

then changing this to `@include icon-content($angle-right-icon);`

Reply:
So that made me think of a similar way to tackle it. I'm wondering if it would be useful or extensible to create a sass file that includes variables for each of the icons that are commonly used on the site. For example, the breadcrumbs component, we could create a variable, for example, $breadcrumb-right-icon: "\f105";. This way, should the icon we use ever need to be updated or changed, we only have to update in this spot, vs having to add the icon to its own stylesheet, then updating the stylesheet where the import is held. Let me know if this makes sense

Reply:
as per our chat, we will name it generically for now, and in the case its repeated a bunhc of times in a file, we will make a local variable